setup.py

```diff
@@ -13,8 +13,6 @@
 
 requirements = ['Click>=6.0', ]
 
-setup_requirements = ['pytest-runner', ]
```

Comment:
pytest-runner suggests it's deprecated and one should use tox (which you already are) https://github.com/pytest-dev/pytest-runner/#deprecation-notice